Modules/Office365DSC/DSCResources/MSFT_SPOAccessControlSettings/MSFT_SPOAccessControlSettings.psm1

```diff
@@ -0,0 +1,349 @@
+            DisplayStartASiteOption      = $SPOAccessControlSettings.DisplayStartASiteOption
+            StartASiteFormUrl            = $SPOAccessControlSettings.StartASiteFormUrl
+            IPAddressEnforcement         = $SPOAccessControlSettings.IPAddressEnforcement
+            IPAddressAllowList           = $SPOAccessControlSettings.IPAddressAllowList
+            IPAddressWACTokenLifetime    = $SPOAccessControlSettings.IPAddressWACTokenLifetime
+            CommentsOnSitePagesDisabled  = $SPOAccessControlSettings.CommentsOnSitePagesDisabled
+            SocialBarOnSitePagesDisabled = $SPOAccessControlSettings.SocialBarOnSitePagesDisabled
+            DisallowInfectedFileDownload = $SPOAccessControlSettings.DisallowInfectedFileDownload
+            ExternalServicesEnabled      = $SPOAccessControlSettings.ExternalServicesEnabled
+            EmailAttestationRequired     = $SPOAccessControlSettings.EmailAttestationRequired
+            EmailAttestationReAuthDays   = $SPOAccessControlSettings.EmailAttestationReAuthDays
+        }
+    }
+    catch
+    {
+        if ($error[0].Exception.Message -like "No connection available")
+        {
+            Write-Verbose "Make sure that you are connected to your SPOService"
+        }
+        return $nullReturn
+    }
+
+}
+function Set-TargetResource
+{
+    [CmdletBinding()]
+    param (
+
+        [Parameter(Mandatory = $true)]
+        [ValidateSet('Yes')]
+        [String]
+        $IsSingleInstance,
+
+        [Parameter()]
+        [System.Boolean]
+        $DisplayStartASiteOption,
+
+        [Parameter()]
+        [System.String]
+        $StartASiteFormUrl,
+
+        [Parameter()]
+        [System.Boolean]
+        $IPAddressEnforcement,
+
+        [Parameter()]
+        [System.String]
+        $IPAddressAllowList,
+
+        [Parameter()]
+        [System.UInt32]
+        $IPAddressWACTokenLifetime,
+
+        [Parameter()]
+        [System.Boolean]
+        $CommentsOnSitePagesDisabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $SocialBarOnSitePagesDisabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $DisallowInfectedFileDownload,
+
+        [Parameter()]
+        [System.boolean]
+        $ExternalServicesEnabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $EmailAttestationRequired,
+
+        [Parameter()]
+        [System.UInt32]
+        $EmailAttestationReAuthDays,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $CentralAdminUrl,
+
+        [Parameter(Mandatory = $true)]
+        [System.Management.Automation.PSCredential]
+        $GlobalAdminAccount
+    )
+
+    Test-PnPOnlineConnection -SPOCentralAdminUrl $CentralAdminUrl -GlobalAdminAccount $GlobalAdminAccount
+
+    $CurrentParameters = $PSBoundParameters
+    $CurrentParameters.Remove("CentralAdminUrl")
+    $CurrentParameters.Remove("GlobalAdminAccount")
+    $CurrentParameters.Remove("Verbose")
```

Comment:
Verbose would never be part of the PSBoundParameters. Therefore it is safe to remove this line